frontend/packages/gitops-plugin/src/components/details/GitOpsServiceDetailsSection.tsx

```diff
@@ -19,60 +20,61 @@ interface GitOpsServiceDetailsSectionProps {
-                          {service.source?.url}
-                        </Label>
-                      </>
-                    </ExternalLink>
-                  ) : (
-                    <div className="odc-gitops-service__details">
-                      {t('gitops-plugin~Service source URL not available')}
-                    </div>
-                  )}
-                </CardBody>
-              </Card>
-            </StackItem>
+            // <StackItem className="odc-gitops-service" key={service.name}>
+            //   <Card>
+            //     <CardTitle className="odc-gitops-service__title co-nowrap">
+            //       {service.workloadKind ? (
+            //         <ResourceLink kind={service.workloadKind} linkTo={false}>
+            //           <Label className="odc-gitops-service__title__label" isTruncated>
+            //             {service.name}
+            //           </Label>
+            //         </ResourceLink>
+            //       ) : (
+            //         <Label className="odc-gitops-service__title__name" isTruncated>
+            //           <span className="co-resource-item__resource-name">{service.name}</span>
+            //         </Label>
+            //       )}
+            //     </CardTitle>
+            //     <CardBody>
+            //       <Label className="co-nowrap odc-gitops-service__image" color="cyan" isTruncated>
+            //         {service.image || <div>{t('gitops-plugin~Image not available')}</div>}
+            //       </Label>
+            //       <Split className="odc-gitops-service__details">
+            //         <SplitItem>
+            //           <div className="odc-gitops-service__pod">{service.podRing}</div>
+            //         </SplitItem>
+            //         <SplitItem className="odc-gitops-service__pr" isFilled>
+            //           {service.commitDetails}
+            //         </SplitItem>
+            //       </Split>
+            //       {service.source?.url ? (
+            //         <ExternalLink
+            //           additionalClassName="odc-gitops-service__url co-truncate co-nowrap"
+            //           href={service.source?.url}
+            //         >
+            //           <>
+            //             {service.source?.icon}&nbsp;
+            //             <Label className="odc-gitops-service__url-label" isTruncated>
+            //               {service.source?.url}
+            //             </Label>
+            //           </>
+            //         </ExternalLink>
+            //       ) : (
+            //         <div className="odc-gitops-service__details">
+            //           {t('gitops-plugin~Service source URL not available')}
+            //         </div>
+            //       )}
+            //     </CardBody>
+            //   </Card>
+            // </StackItem>
+            <StackItem key={service.name}>{service.commitDetails}</StackItem>
```

Comment:
I take it this is a WIP section given the state... but `StackItem` probably shouldn't be removed from the usage of `Stack`. Ie, it should probably be either named-as (ie `GitOpsReasonForItStackedItem`) or be in the same component (more ideal).